notices.json

```diff
@@ -2,9 +2,6 @@
     {
     "id": "use_case_survey",
     "conditions": {
-      "audience": "all",
-      "clientType": "all",
-      "instanceType": "all",
```

Comment:
Removed redundant conditions that default to "all" when not explicitly stated.

Reply:
Thanks!